### PR TITLE
[#9480] Add snapshot test for instructor search page

### PR DESCRIPTION
**Pull request by rrtheonlyone**
Part of #9480

Added a few simple test cases for the instructor search page. 

1. No search results
2. Have search results

### CODE REVIEW

File: src/web/app/pages-instructor/instructor-search-page/__snapshots__/instructor-search-page.component.spec.ts.snap [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`InstructorSearchPageComponent should snap with a student table 1`] = `
+<tm-instructor-search-page
+  fbSessionDataTables={[Function Array]}
+  httpRequestService={[Function HttpRequestService]}
+  route={[Function ActivatedRoute]}
+  searchKey=""
```
Comment (monmanuela) [inline src/web/app/pages-instructor/instructor-search-page/__snapshots__/instructor-search-page.component.spec.ts.snap]: You should populate the `searchKey` to check that it's rendered correctly

File: src/web/app/pages-instructor/instructor-search-page/__snapshots__/instructor-search-page.component.spec.ts.snap [hunk 2]
```diff
@@ -0,0 +1,35 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`InstructorSearchPageComponent should snap with a student table 1`] = `
+<tm-instructor-search-page
+  fbSessionDataTables={[Function Array]}
```
Comment (monmanuela) [inline src/web/app/pages-instructor/instructor-search-page/__snapshots__/instructor-search-page.component.spec.ts.snap]: You should populate this too for one of the test cases

File: src/web/app/pages-instructor/instructor-search-page/__snapshots__/instructor-search-page.component.spec.ts.snap [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  httpRequestService={[Function HttpRequestService]}
+  route={[Function ActivatedRoute]}
+  searchKey=""
+  statusMessageService={[Function StatusMessageService]}
+  studentTables={[Function Array]}
+  user="undefined"
```
Comment (monmanuela) [inline src/web/app/pages-instructor/instructor-search-page/__snapshots__/instructor-search-page.component.spec.ts.snap]: This too

File: src/web/app/pages-instructor/instructor-search-page/__snapshots__/instructor-search-page.component.spec.ts.snap (hunk 1 above)
Comment (rrtheonlyone) [reply src/web/app/pages-instructor/instructor-search-page/__snapshots__/instructor-search-page.component.spec.ts.snap]: Yup your right! Added

File: src/web/app/pages-instructor/instructor-search-page/__snapshots__/instructor-search-page.component.spec.ts.snap (hunk 2 above)
Comment (rrtheonlyone) [reply src/web/app/pages-instructor/instructor-search-page/__snapshots__/instructor-search-page.component.spec.ts.snap]: Interestingly this is not being used in the html template itself

Comment (rrtheonlyone) [reply src/web/app/pages-instructor/instructor-search-page/__snapshots__/instructor-search-page.component.spec.ts.snap]: I think that part is still under migration? @wkurniawan07

File: src/web/app/pages-instructor/instructor-search-page/__snapshots__/instructor-search-page.component.spec.ts.snap (hunk 3 above)
Comment (rrtheonlyone) [reply src/web/app/pages-instructor/instructor-search-page/__snapshots__/instructor-search-page.component.spec.ts.snap]: this value also does not seem to have a use

Comment (monmanuela) [reply src/web/app/pages-instructor/instructor-search-page/__snapshots__/instructor-search-page.component.spec.ts.snap]: Ah yup don't need

**Comment by rrtheonlyone** [comment]
Hi @monmanuela / @AyushChatto , this page is still under migration. I have done a bare bones snapshot test for the currently available components. Do see if it is okay.

File: src/web/app/pages-instructor/instructor-search-page/instructor-search-page.component.spec.ts [hunk 4]
```diff
@@ -41,4 +42,64 @@ describe('InstructorSearchPageComponent', () => {
...
+        ],
+      },
+    ];
+
+    fixture.detectChanges();
+    expect(fixture).toMatchSnapshot();
+  });
+
+  it('should snap with a feedback session table', () => {
+
```
Comment (AyushChatto) [inline src/web/app/pages-instructor/instructor-search-page/instructor-search-page.component.spec.ts]: Just one qn - why is this one blank?

Comment (rrtheonlyone) [reply src/web/app/pages-instructor/instructor-search-page/instructor-search-page.component.spec.ts]: Sry, this was a KIV. I have removed it!

**Pull request merged**